docs/user/openstack/README.md

```diff
@@ -176,68 +216,132 @@ Create and associate a floating IP to the ingress port:
 openstack floating ip create --port <ingress port id> <external network>
 ```
 
-Add A record in your dns for *apps. in your DNS:
-
-```
-*.apps.example.shiftstack.com  IN  A  <ingress FIP>
-```
-OR add A record in `/etc/hosts`:
-
-```
-    <ingress FIP> console-openshift-console.apps.example.shiftstack.com
-```
-
 ### Without Floating IPs
 
 If you don't want to pre-create a floating IP address, you will still want to create the API DNS record or the installer will fail waiting for the API.
 Without the floating IP, you won't know the right IP address of the server ahead of time, so you will have to wait for it to come up and create the DNS records then:
 
-    $ watch openstack server list
+```sh
+watch openstack server list
+```
 
 Wait for the `<cluster name>-api` server comes up and you can make your changes then.
```

Comment:
We've removed the service VM, users can wait a long time for `<cluster name>-api` :)